bld.bat

```diff
@@ -0,0 +1,2 @@
+"%PYTHON%" setup.py install --single-version-externally-managed --record=record.txt
+if errorlevel 1 exit 1
```

Comment:
I am not familiar with this .bat file. Is it something we need or could we have it in `.gitignore`?

Reply:
This file is used to build the Conda package for Windows users. For details on building the Conda package, see below.